sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample12_RecognizeBusinessCardsFromFile.cs

```diff
@@ -73,40 +73,6 @@ public async Task RecognizeBusinessCardsFromFile()
                         }
                     }
 
-                    FormField jobTitlesFields;
-                    if (businessCard.Fields.TryGetValue("JobTitles", out jobTitlesFields))
-                    {
-                        if (jobTitlesFields.Value.ValueType == FieldValueType.List)
-                        {
-                            foreach (FormField jobTitleField in jobTitlesFields.Value.AsList())
-                            {
-                                if (jobTitleField.Value.ValueType == FieldValueType.String)
-                                {
-                                    string jobTitle = jobTitleField.Value.AsString();
-
-                                    Console.WriteLine($"  Job Title: '{jobTitle}', with confidence {jobTitleField.Confidence}");
-                                }
-                            }
-                        }
-                    }
-
-                    FormField departmentFields;
-                    if (businessCard.Fields.TryGetValue("Departments", out departmentFields))
-                    {
-                        if (departmentFields.Value.ValueType == FieldValueType.List)
-                        {
-                            foreach (FormField departmentField in departmentFields.Value.AsList())
-                            {
-                                if (departmentField.Value.ValueType == FieldValueType.String)
-                                {
-                                    string department = departmentField.Value.AsString();
-
-                                    Console.WriteLine($"  Department: '{department}', with confidence {departmentField.Confidence}");
-                                }
-                            }
-                        }
```

Comment:
another question of curiosity: why did you remove some fields and how did you decide which fields to remove? I assume you removed them since the sample is so long, but am just curious

Reply:
Yes. too long and the readme was getting sooooo long. I am following what Java did, where for the readme snippets show only a few but for the real BusinessCards I am leaving them all so the user can just copy paste